nimble/controller/src/ble_ll_sched.c

```diff
@@ -786,6 +786,22 @@ ble_ll_sched_slave_new(struct ble_ll_conn_sm *connsm)
     return rc;
 }
 
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_EXT_ADV)
+static inline uint32_t
+usecs_to_ticks_fast(uint32_t usecs)
+{
+    uint32_t ticks;
+
+    if (usecs <= 31249) {
+        ticks = (usecs * 137439) / 4194304;
```

Comment:
Not even "John Carmack's inverse square" style comment ?:)

Reply:
that's too obvious ;-)